aioxmpp/ibr/xso.py

```diff
@@ -94,6 +99,11 @@ class Query(xso.XSO):
         default=None,
     )
 
+    old_password = xso.ChildText(
+        (namespaces.xep0077_in_band, "old_password"),
+        default=None,
+    )
```

Comment:
This does not exist in XEP-0077.

Reply:
Looking at section 13.4.3 "Change Password" it looks like the field could exist, doesn't it?